src/reconcile/reconcile_private.h

```diff
@@ -184,6 +184,13 @@ extern int __wti_rec_dictionary_lookup(WT_SESSION_IMPL *session, WT_RECONCILE *r
   WTI_REC_DICTIONARY **dpp) WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
 extern int __wti_rec_hs_clear_on_tombstone(WT_SESSION_IMPL *session, WT_RECONCILE *r,
   uint64_t recno, WT_ITEM *rowkey, bool reinsert) WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
+extern int __wti_rec_hs_delete_key(WT_SESSION_IMPL *session, WT_CURSOR *hs_cursor,
```

Comment:
You have to remove the `reconcile_private.h` inclusion in the `reconcile.h` file. And also move all the structures and macros that are no longer needed in them.

Reply:
I have moved the structures `__wti_delete_hs_upd` and `__wti_reconcile` to `reconcile_private.h` in fac78b67dec968b8e614a4949151973e0b747f03.  I cannot remove the inclusion of `reconcile_private.h` from `reconcile.h` because many functions defined in the reconcile module calls the private functions.